src/wmtk/invariants/FrozenVertexInvariant.cpp

```diff
@@ -0,0 +1,41 @@
+#include "FrozenVertexInvariant.hpp"
+#include <wmtk/Mesh.hpp>
+
+namespace wmtk::invariants {
+FrozenVertexInvariant::FrozenVertexInvariant(
```

Comment:
If this invariant is only used in your component, please move it to your component. Only move it to the src folder, if it is used in multiple components.

Reply:
will do in the next pr